Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = (props) => {
+    let embedTitle = "Get started with React";
+    let embedContent = "React makes it painless to create interactive UIs. Design simple views for each state in your application."
+    let embedLink = "reactjs.org"
```

Comment:
should these be `let`s or `const`s?

Reply:
Well, if I'm going to change them later, shouldn't they be `let`? Maybe I'm wrong.